src/backend/distributed/commands/aggregate.c

```diff
@@ -0,0 +1,62 @@
+/*-------------------------------------------------------------------------
+ *
+ * aggregate.c
+ *    Commands for distributing AGGREGATE statements.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "distributed/commands.h"
+#include "distributed/commands/utility_hook.h"
+#include "distributed/deparser.h"
+#include "distributed/listutils.h"
+#include "distributed/metadata_sync.h"
+#include "distributed/metadata/distobject.h"
+#include "nodes/parsenodes.h"
+
+/*
+ * PreprocessDefineAggregateStmt is called during the planning phase for
+ * CREATE AGGREGATE ..
+ */
+List *
+PreprocessDefineAggregateStmt(Node *node, const char *queryString,
+							  ProcessUtilityContext processUtilityContext)
+{
+	QualifyTreeNode((Node *) node);
+
+	return NIL;
```

Comment:
Can you add checks whether we should propagate create aggregate statement here? If so we should ensure we are on the coordinator and can switch back to sequential mode here.